src/main/webapp/app/layouts/profiles/page-ribbon.component.ts

```diff
@@ -22,7 +22,8 @@ import { ProfileInfo } from './profile-info.model';
 
 @Component({
     selector: 'jhi-page-ribbon',
-    template: `<div class="ribbon" *ngIf="ribbonEnv"><a href="">{{ribbonEnv}}</a></div>`,
+    template: `<div class="devRibbon" *ngIf="ribbonEnv"><a href="">{{ribbonEnv}}</a></div>
+    <div class="forkRibbon" ><a href="https://github.com/jhipster/jhipster-online">Fork me on GitHub</a></div>`,
```

Comment:
I dont like having this as a ribbon, since its a web app and not a doc site, we can rather have this as a link in the footer

Reply:
Let me check - I'm the one who wanted the ribbon, it should be on the bottom, to do some adverstisement about the Open Source project. Then, I wonder if that works fine in prod mode.